portal/app/[locale]/stake/_hooks/useWalletBalance.ts

```diff
@@ -0,0 +1,29 @@
+import { useQueries } from '@tanstack/react-query'
+import { useHemiClient } from 'hooks/useHemiClient'
+import { useStakeTokens } from 'hooks/useStakeTokens'
+import { StakeToken } from 'types/stake'
+import { useAccount } from 'wagmi'
+
+export const useWalletPositions = function () {
```

Comment:
I think the name of this file should match the hook name. So either this hook needs to be renamed to `useWalletBalance` , or the file to `useWalletPositions`.

I think I prefer `useWalletBalances` (note the plural) everywhere

Reply:
fixed